tests/compatibility-test.py

```diff
@@ -1,150 +1,19 @@
 #!/usr/bin/env python
 import logging
-import os
-import tempfile
 import unittest
-import subprocess
-from string import Template
-
 import docker
-import time
 
-ray_version = '1.9.0'
-ray_image = "rayproject/ray:1.9.0"
+from kuberay_utils.utils import *
```

Comment:
\* imports are considered bad style for various reasons

Would you mind either
- Importing each utility used
OR
- Doing from `kuberay_utils import utils` and prefixing each utility with `utils.`